src/main/java/edu/ucsb/cs156/example/controllers/UCSBOrganizationController.java

```diff
@@ -29,13 +29,13 @@
 public class UCSBOrganizationController extends ApiController {
 
     @Autowired
-    UCSBOrganizationRepository ucsbOrganizationRepository;
+    UCSBOrganizationRepository UCSBOrganizationRepository;
```

Comment:
like here,

1. now isn't in camel case, the standard coding style for Java
2. it has the exact same name as its class
3. this was from team02, there is zero point in touching it